psl/psl/src/lib.rs

```diff
@@ -1,5 +1,47 @@
 #![doc = include_str!("../README.md")]
 #![deny(rust_2018_idioms, unsafe_code, missing_docs)]
 
+pub use builtin_psl_connectors as builtin_connectors;
 pub use dml::{self, lift, render_datamodel_and_config_to_string, render_datamodel_to_string};
-pub use psl_core::*;
+pub use psl_core::{
+    common,
+    datamodel_connector,
+    diagnostics::{self, Diagnostics},
+    is_reserved_type_name,
+    mcf::config_to_mcf_json_value as get_config,
+    mcf::{generators_to_json, render_sources_to_json}, // for tests
+    parser_database::{self, SourceFile},
+    reformat,
+    Configuration,
+    Datasource,
+    Generator,
+    StringFromEnvVar,
+    ValidatedSchema,
+};
+
+/// The implementation of the CLI getConfig() utility and its JSON format.
+pub mod get_config {
+    pub use psl_core::mcf::{config_to_mcf_json_value as get_config, *};
+}
+
+/// Parses and validate a schema, but skip analyzing everything except datasource and generator
+/// blocks.
+pub fn parse_configuration(schema: &str) -> Result<Configuration, Diagnostics> {
+    psl_core::parse_configuration(schema, builtin_connectors::BUILTIN_CONNECTORS)
+}
+
+/// Parse and analyze a Prisma schema.
+pub fn parse_schema_parserdb(file: impl Into<SourceFile>) -> Result<ValidatedSchema, String> {
```

Comment:
```suggestion
pub fn parse_schema(file: impl Into<SourceFile>) -> Result<ValidatedSchema, String> {
```

Reply:
I didn't do it to avoid a big diff, but I agree it's nicer.

Reply:
I'll do it before merging.